modules/nominees-election/src/lib.rs

```diff
@@ -132,16 +162,18 @@ pub mod module {
 		#[pallet::constant]
 		type MaxUnlockingChunks: Get<u32>;
 		type RelaychainValidatorFilter: Contains<Self::NomineeId>;
+		type MaxNominees: Get<u32>;
```

Comment:
```suggestion
		#[pallet::constant]
		type MaxNominees: Get<u32>;
```

Reply:
can we just use `NominateesCount` instead of `MaxNominees`?

Reply:
I was originally thinking that we may want the flexibility not to have them connected but after thinking it through I think it make sense to have them be the same.